pom.xml

```diff
@@ -45,6 +45,16 @@
         <artifactId>jmh-generator-annprocess</artifactId>
         <version>${version.jmh}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.maven.shared</groupId>
+        <artifactId>maven-invoker</artifactId>
+        <version>2.2</version>
+      </dependency>
+      <dependency>
+        <groupId>org.jbpm</groupId>
+        <artifactId>perf-test-common</artifactId>
+        <version>${kie.bom.version}</version>
+      </dependency>
```

Comment:
This part should be in the `jbpm-benchmarks/pom.xml` I think as this is only relevant to jbpm-benchmarks now. And also its version should ideally be `${project.version}` as it is coupled with this project's version rather than with kie version, i.e. if versioning changes in the future, it is more obvious that this module should follow the same pattern.